tools/metactl/src/main.rs

```diff
@@ -72,6 +79,14 @@ async fn main() -> anyhow::Result<()> {
 
     eprintln!("raft_config: {}", pretty(raft_config)?);
 
+    // export from grpc api if raft is running
+    let addr = raft_config.raft_api_addr().await?.to_string();
+    if config.export && service_is_running(&addr).await? {
+        eprintln!("export meta from: {}", &config.grpc_api_address);
+        grpc::export_meta(&config.grpc_api_address).await?;
```

Comment:
Why not just check `grpc_api_address` but `raft_api_addr()`?

BTW it exports from a running metasrv if and only if `grpc_api_address` is specified.

Reply:
Should use `grpc_api_address` indeed.
What if `metasrv` is running and `grpc_api_address` is not specified.

Reply:
If  `grpc_api_address` is invalid metasrv won't run at all.

Besides, if `grpc_api_address` is invalid, there is no way to export data from it.

Reply:
Updated, and a basic test.